lib/Email/Folder/Search.pm

```diff
@@ -25,10 +25,10 @@ Search email from mailbox file. This module is mainly to test that the emails ar
 
 use strict;
 use warnings;
-use NEXT;
 use Encode qw(decode);
 use Scalar::Util qw(blessed);
 use base 'Email::Folder';
+use Class::C3;
```

Comment:
My fault - I should have suggested the core module instead, which is [mro](http://perldoc.perl.org/mro.html).

Replacing this one line with:

```
use mro;
```

should be enough to fix test failures on all versions of perl5.10+.